testvectors/json/test-boot-seed-invalid-long.json

```diff
@@ -0,0 +1,24 @@
+{
+  "eat-profile": "http://arm.com/psa/2.0.0",
```

Comment:
Quick Clarification on all the below samples.

Here we say "eat-profile" is PSA 2.0

and on the last key-val we say:
"psa-profile": "PSA_IOT_PROFILE_1"

This is very confusing, while testing other invalidities like boot-seed etc.

Reply:
that's very handy because it allows us to test scenarios that are common to P1 and P2 with the same test vector.  the alternative would be to duplicate all the common test vectors with just the profile claim changed.

Reply:
But these are inputs to encoding right? It should have just the error bit incorrect, like boot seed but rest all inputs should be correct, from the sample right ?

Ideally we can create two sample file set one for P1 profile and the other for P2 profile?

We can test few invalidities with P1 and few invalidities with P2. right ?

Reply:
> But these are inputs to encoding right?

no these are inputs for decoding (see loadJSONTestVectorFromFile)

Reply:
May be you explain me this on Monday. The logic in the file loads these claims from loadJSONTestVectorFromFile  and 
then calls Validate on these claims ? its a bit confusing...

Reply:
resolved f2f